### DISCUSSION TITLE
Import only certain Plex users

### DISCUSSION THREAD

**Discussion by ghost** [sct/overseerr#1296] [Feature Requests]
Just an idea to somehow get a list of Plex Users and choose who you want to import into Overseerr as opposed to importing everyone from Plex. 

Or even an exclude list

Love the project btw, keep up the great work!

**Comment by liamcottle** [comment]
+1, was about to ask the same.

It would be ideal to be able to select specific users to give access to overseerr through Plex OAuth, but without importing everyone from Plex.

This would also make it easier to know who has access to overseerr, since only the accounts with access would be listed.

At the moment, you would need to either create local users, or import all plex users and then disable access for numerous accounts.
